docs/sources/dashboards/dashboard-public/index.md

```diff
@@ -2,6 +2,11 @@
 aliases:
   - ../features/dashboard/dashboards/
   - dashboard-manage/
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
```

Comment:
@evictorero  or @juanicabanas - can you confirm? it's only available in Cloud, right? 
```suggestion
    - enterprise
    - oss
```